src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -488,7 +496,7 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
                   display: 'flex',
                   flexGrow: 1,
                   height: '100%',
-                  backgroundColor: theme.palette.gray[50],
```

Comment:
I think these colors might actually be different.
Could you update this to just the string 'gray.50' please?

sx props should be able to get colors once again :)

Reply:
yeah the compiler is complaining about these for some reason. They do crash my build, will revert